plugins/modules/zos_job_submit.py

```diff
@@ -660,16 +672,20 @@ def submit_src_jcl(module, src, timeout=0, hfs=True, volume=None, start_time=tim
                 job_listing_rc = jobs.listing(job_submitted.id)[0].rc
                 job_listing_status = jobs.listing(job_submitted.id)[0].status
 
-    # ZOAU throws a ZOAUException when the job sumbission fails, not when the
-    # JCL is non-zero, for non-zero JCL RCs that is caught in the job_output
-    # processing
+    # ZOAU throws a ZOAUException when the job sumbission fails thus there is no
+    # JCL RC to share with the user, if there is a RC, that will be processed
+    # in the job_output parser.
     except ZOAUException as err:
         result["changed"] = False
         result["failed"] = True
         result["stderr"] = str(err)
-        result["msg"] = ("Unable to submit job {0}, a job sumission has returned "
-                         "a non-zero return code, please review the standard error "
-                         "and contact a system administrator.".format(src))
+        result["duration"] = duration
+        result["job_id"] = job_submitted.id if job_submitted else None
+        result["msg"] = ("Unable to submit job {0}, the job submission has failed. "
+                         "Without the job id, the error can not be determined, "
```

Comment:
Same comment as other ticket: This line is an uncapitalized, run-on.  Without should be capitalized, and determined should end the sentence.  The next line, Consider should be capitalized.  Not a deal-breaker, but I wanted you to see it.